### PR TITLE
feat : password all screens

### PR DESCRIPTION
**Pull request by p-r1sm**
## Screenshots
| Page | SS |
|--|--|
| Forget Password | image|
| OTP Interface | image|
| Reset Password | image|
| Password Updated| image|

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -15,6 +16,8 @@ const App = () => {
         <Route path="/" element={<LandingPage />} />
         <Route path="/login" element={<Login />} />
         <Route path="/register" element={<Register />} />
+        <Route path="/" element={<Home />} />
```
Comment (SaurabhK05) [inline src/App.js]: LandingPage and Home component are same, please remove Home

File: src/Pages/Home.jsx
Comment (SaurabhK05) [inline src/Pages/Home.jsx:1]: remove this file

Comment (SaurabhK05) [reply src/Pages/Home.jsx:1]: Please take latest from integration branch

File: src/App.js (hunk 1 above)
Comment (p-r1sm) [reply src/App.js]: Done

File: src/Pages/Home.jsx
Comment (p-r1sm) [reply src/Pages/Home.jsx:1]: Removed

File: src/Pages/forgotPassword/styles.css [hunk 2]
```diff
@@ -0,0 +1,120 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins&display=swap");
```
Comment (SaurabhK05) [inline src/Pages/forgotPassword/styles.css]: This dependency is already added in index.html

Comment (p-r1sm) [reply src/Pages/forgotPassword/styles.css]: removed

**Comment by SaurabhK05** [comment]
Your deployment is failing try to fix it.
image

**Comment by SaurabhK05** [comment]
> Your deployment is failing try to fix it. image

Issue fixed

File: src/components/Footer/Footer.jsx
Comment (SaurabhK05) [inline src/components/Footer/Footer.jsx:1]: Please keep integration Footer changes then apply your changes. this file will going to override Footer

File: src/components/Header/Header.jsx
Comment (SaurabhK05) [inline src/components/Header/Header.jsx:1]: Same for this file as well

Comment (SaurabhK05) [reply src/components/Header/Header.jsx:1]: Please integrate forget page with login/signup page

Comment (SaurabhK05) [reply src/components/Header/Header.jsx:1]: Rest thing looks good

Comment (SaurabhK05) [reply src/components/Header/Header.jsx:1]: Add screenshot of new page/component in description, please refer this PR https://github.com/pesto-students/team1-manish-team1-manish/pull/9

Comment (p-r1sm) [reply src/components/Header/Header.jsx:1]: Okay, adding

File: src/components/Footer/Footer.jsx
Comment (SaurabhK05) [reply src/components/Footer/Footer.jsx:1]: Issue fixed

File: src/components/Header/Header.jsx
Comment (SaurabhK05) [reply src/components/Header/Header.jsx:1]: Have added in description

**Review by SaurabhK05 — COMMENTED** [review]
Looks good to me @RyuzaKiRa and @PranavSrisrimal, Kindly approve this PR

**Review by hritiksingh13 — APPROVED** [review]
LGTM

**Pull request merged**